TheEthicalHackerCup/Assets/Resources/Ransomware2.xml

```diff
@@ -1,20 +1,39 @@
 <Quiz>
-	<Slide Name="SLIDE 1">
-		<Content Type="Learning.InfoContent">Welcome</Content>
+	<Slide Name="Welcome to the second ransomware quiz">
+		<Content Type="Learning.InfoContent">This quiz will focus on avoiding and detecting ransomware</Content>
 	</Slide>
-	<Slide Name="SLIDE 2">
-		<Content Type="Learning.CheckBox">
-			<Correct>1</Correct>
-			<Option>Hello</Option>
-			<Option>Bruh</Option>
+	<Slide Name="Avoidance">
+		<Content Type="Learning.InfoContent">Avoiding ransomware, and malware in general is simple from a users point of view. As long as their system stays up to date and the only things being downloaded are from trusted sources, this in most cases is suffient.</Content>
+	</Slide>
+	<Slide Name="Detection">
+		<Content Type="Learning.InfoContent">Antivirus is the second line of defense. If someone installs malware, antivirus is a type of program designed to detect and prevent the malware from running.</Content>
+	</Slide>
+	<Slide Name="Detection">
+		<Content Type="Learning.InfoContent">Antivirus used to classify files via signiture detection. This is where it would analyze the contents of the file and see if it recognizes any patterns of known viruses. Modern antiviruses work through heuristic detection alongside signiture detection. Heuristic detection is where the viruses are detected by running the program and analyzing its behaviour within an environment where the virus wont cause any damage.</Content>
+	</Slide>
```

Comment:
![image](https://user-images.githubusercontent.com/72239525/225659493-4230dba4-8808-4235-9150-44f2759a342e.png)
This would fit better as 2 separate slides